pkg/api/v20220904/clustermanager_validatestatic.go

```diff
@@ -0,0 +1,8 @@
+package v20220904
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// TODO Implement the static validator
```

Comment:
can this file be deleted for now and a story created to implement it? (to avoid committing commented out code to master)

Reply:
https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/15533307/